lib/utils/dataflow/src/index.js

```diff
@@ -319,12 +327,27 @@ const postOutput = function *(odoc,
   }
 };
 
+// Check if any of the docs have errors in it
+const validateDocs = (docs) => find(docs, (doc) => doc.error) ? [] : docs;
+
 // Post a list of output docs to the configured sink service
 const postOutputs = function *(odocs,
   shost, spartition, spost, authentication) {
-  yield tmap(odocs, function *(odoc, i, l) {
```

Comment:
I actually change this back to tmap in later commits in this pull request(my bad for not stashing them). I was trying out treduce because I thought the doc would just be a single doc everytime after the change in collector, but aggregator proves me wrong.